status/flowcells.py

```diff
@@ -416,10 +416,19 @@ def get(self, query):
             self.write(t.generate(gs_globals=self.application.gs_globals, user=self.get_current_user(), readsdata=ordereddata, query=query))
         else:
             xfc_view = self.application.x_flowcells_db.view("samples/lane_clusters", reduce=False)
+            bioinfo_view = self.application.bioinfo_db.view("latest_data/sample_id")
             for row in xfc_view[query:"{}Z".format(query)]:
                 if not row.key in data:
                     data[row.key]=[]
                 data[row.key].append(row.value)
+            #To check if sample is failed on lane level
+            for row in bioinfo_view:
```

Comment:
It would be better to loop through `data` and get the relevant samples from `bioinfo_view` rather than loop through the entire view. If its difficult to get individual samples from `bioinfo_view`, a new view where we can filter for samples makes more sense.

Reply:
It's not so much the samples, each entry in both views is on sample level, as they should be I think since we have to get the samples, flowcell, and lane per sample and also per view to compare them and see if they match. 

Not quite sure how to do that without the looping, we still have to do some, but I can see if I can reduce it somehow 🤔 👍